modules/virt-running-ssp-pipeline-cli.adoc

```diff
@@ -10,87 +10,47 @@ Use a `PipelineRun` resource to run the example pipelines. A `PipelineRun` objec
 
 .Procedure
 
-. To run the Windows 10 installer pipeline, create the following `PipelineRun` manifest:
+. To run the Microsoft Windows 11 installer pipeline, create the following `PipelineRun` manifest:
 +
-[source,yaml]
+[source,yaml,subs="attributes+"]
 ----
-apiVersion: tekton.dev/v1beta1
+apiVersion: tekton.dev/v1
 kind: PipelineRun
 metadata:
-  generateName: windows10-installer-run-
+  generateName: windows11-installer-run-
   labels:
-    pipelinerun: windows10-installer-run
+    pipelinerun: windows11-installer-run
 spec:
-  params:
-  - name: winImageDownloadURL
-    value: <link_to_windows_10_iso> <1>
-  pipelineRef:
-    name: windows10-installer
-  taskRunSpecs:
-    - pipelineTaskName: copy-template
-      serviceAccountName: copy-template-task
-    - pipelineTaskName: modify-vm-template
-      serviceAccountName: modify-vm-template-task
-    - pipelineTaskName: create-vm-from-template
-      serviceAccountName: create-vm-from-template-task
-    - pipelineTaskName: wait-for-vmi-status
-      serviceAccountName: wait-for-vmi-status-task
-    - pipelineTaskName: create-base-dv
-      serviceAccountName: modify-data-object-task
-    - pipelineTaskName: cleanup-vm
-      serviceAccountName: cleanup-vm-task
-  status: {}
-----
-<1> Specify the URL for the Windows 10 64-bit ISO file. The product language must be English (United States).
-
-. Apply the `PipelineRun` manifest:
-+
-[source,terminal]
-----
-$ oc apply -f windows10-installer-run.yaml
-----
-
-. To run the Windows 10 customize pipeline, create the following `PipelineRun` manifest:
-+
-[source,yaml]
-----
-apiVersion: tekton.dev/v1beta1
-kind: PipelineRun
-metadata:
-  generateName: windows10-customize-run-
-  labels:
-    pipelinerun: windows10-customize-run
-spec:
-  params:
-    - name: allowReplaceGoldenTemplate
-      value: true
-    - name: allowReplaceCustomizationTemplate
-      value: true
-  pipelineRef:
-    name: windows10-customize
-  taskRunSpecs:
-    - pipelineTaskName: copy-template-customize
-      serviceAccountName: copy-template-task
-    - pipelineTaskName: modify-vm-template-customize
-      serviceAccountName: modify-vm-template-task
-    - pipelineTaskName: create-vm-from-template
-      serviceAccountName: create-vm-from-template-task
-    - pipelineTaskName: wait-for-vmi-status
-      serviceAccountName: wait-for-vmi-status-task
-    - pipelineTaskName: create-base-dv
-      serviceAccountName: modify-data-object-task
-    - pipelineTaskName: cleanup-vm
-      serviceAccountName: cleanup-vm-task
-    - pipelineTaskName: copy-template-golden
-      serviceAccountName: copy-template-task
-    - pipelineTaskName: modify-vm-template-golden
-      serviceAccountName: modify-vm-template-task
-status: {}
-----
-
+    params:
+    -   name: winImageDownloadURL
+        value: ${WIN_IMAGE_DOWNLOAD_URL} <1>
+    -   name: acceptEula
+        value: false <2>
```

Comment:
See the comment above regarding commenting out the callout in YAML files. If this occurs again, I won't mention it in the future. Please apply this comment throughout.